docs/ref/contrib/sitemaps.txt

```diff
@@ -249,6 +253,17 @@ Sitemap class reference
         be generated using all of your :setting:`LANGUAGES`. The default is
         ``False``.
 
+    .. function:: Sitemap.get_latest_lastmod()
+
+        **Optional.** A method that returns the latest value returned by :attr:`~Sitemap.lastmod()`.
+
+        By default :meth:`~Sitemap.get_latest_lastmod()` returns:
+
+        * If :attr:`~Sitemap.lastmod` is an attribute: :attr:`~Sitemap.lastmod`.
+        * If :attr:`~Sitemap.lastmod` is a method: ``None``.
+
+        .. versionadded:: 1.10
```

Comment:
put what exactly, where exactly?

I copied the format/layout of `attribute:: Sitemap.priority` above (line 216).